ddtrace/appsec/_iast/_ast/ast_patching.py

```diff
@@ -28,6 +28,47 @@
 # Prefixes for modules where IAST patching is allowed
 IAST_ALLOWLIST: Tuple[Text, ...] = ("tests.appsec.iast.",)
 IAST_DENYLIST: Tuple[Text, ...] = (
+    "altgraph.",
+    "dipy.",
+    "black.",
+    "mypy.",
+    "mypy_extensions.",
+    "autopep8.",
+    "pycodestyle.",
+    "pydicom.",
+    "PyInstaller.",
+    "pystray.",
+    "contourpy.",
+    "cx_Logging.",
+    "dateutil.",
+    "pytz.",
+    "wcwidth.",
+    "win32ctypes.",
+    "Xlib.",
+    "cycler.",
+    "Cython.",
```

Comment:
I believe this has to be lowercase, as we're doing `lower()` to the original module before comparing

Reply:
Done, thanks!